packages/salesforcedx-vscode-core/src/commands/templates/lightningGenerateLwc.ts

```diff
@@ -34,32 +35,42 @@ export class LibraryLightningGenerateLwcExecutor extends LibraryBaseTemplateComm
   public telemetryName = 'lightning_generate_lwc';
   public metadataTypeName = LWC_TYPE;
   public templateType = TemplateType.LightningComponent;
+  private templateOptions: LightningComponentOptions | undefined;
```

Comment:
The `LightningComponentOption` requires this [PR](https://github.com/forcedotcom/salesforcedx-templates/pull/590) to pass compilation.